src/libfabric/context.hpp

```diff
@@ -61,6 +65,59 @@ class context_impl : public context_base
 
     inline controller_type* get_controller() /*const */ { return m_controller.get(); }
     const char*             get_transport_option(const std::string& opt);
+
+    void progress() { get_controller()->poll_for_work_completions(nullptr); }
+
+    bool cancel_recv(detail::shared_request_state* s)
+    {
+        return false;
```

Comment:
Cut'n'paste issue with this function - Do you want to return false? Seems like a duplicate of the other cancel function, but specialized for a shared receive.

Reply:
this statement should go. yes, will fix